lib/Parser/rterrors.h

```diff
@@ -21,84 +21,84 @@
+RT_ERROR_MSG(JSERR_OLEFileNotFound, 432,    "",   "File name or class name not found during Automation operation", kjstError, 0)
+RT_ERROR_MSG(JSERR_OLENoPropOrMethod, 438,    "Object doesn't support property or method '%s'",    "Object doesn't support this property or method", kjstTypeError, 0)
+//RT_ERROR_MSG(JSERR_OLEAutomationError, 440,    "",    "Automation error", kjstError, 0)
+//RT_ERROR_MSG(JSERR_LostTLB, 442,    "",   "Connection to type library or object library for remote process has been lost. Press OK for dialog to remove reference.")
+//RT_ERROR_MSG(JSERR_OLENoDefault, 443,    "",  "Automation object does not have a default value")
+RT_ERROR_MSG(JSERR_ActionNotSupported, 445,    "",    "Object doesn't support this action", kjstTypeError, 0)
+RT_ERROR_MSG(JSERR_NamedArgsNotSupported, 446,    "", "Object doesn't support named arguments", kjstError, 0)
+RT_ERROR_MSG(JSERR_LocaleSettingNotSupported, 447,    "", "Object doesn't support current locale setting", kjstError, 0)
+RT_ERROR_MSG(JSERR_NamedParamNotFound, 448,    "",    "Named argument not found", kjstError, 0)
+RT_ERROR_MSG(JSERR_ParameterNotOptional, 449,    "Argument to the function '%s' is not optional",  "Argument not optional", kjstTypeError, 0)
+RT_ERROR_MSG(JSERR_FuncArityMismatch, 450,    "", "Wrong number of arguments or invalid property assignment", kjstTypeError, 0)
+RT_ERROR_MSG(JSERR_NotEnum, 451,    "",   "Object not a collection", kjstTypeError, 0)
+//RT_ERROR_MSG(JSERR_InvalidOrdinal, 452,    "",    "Invalid ordinal")
+RT_ERROR_MSG(JSERR_InvalidDllFunctionName, 453,    "",    "Specified DLL function not found", kjstError, 0)
+//RT_ERROR_MSG(JSERR_CodeResourceNotFound, 454,    "",  "Code resource not found")
+//RT_ERROR_MSG(JSERR_CodeResourceLockError, 455,    "", "Code resource lock error")
+//RT_ERROR_MSG(JSERR_DuplicateKey, 457,    "",  "This key is already associated with an element of this collection")
+RT_ERROR_MSG(JSERR_InvalidTypeLibVariable, 458,    "",    "Variable uses an Automation type not supported in JavaScript", kjstTypeError, 0)
+RT_ERROR_MSG(JSERR_ServerNotFound, 462, "", "The remote server machine does not exist or is unavailable", kjstError, 0)
+//RT_ERROR_MSG(JSERR_InvalidPicture, 481,    "",    "Invalid picture")
+
+//RT_ERROR_MSG(JSERR_CantAssignTo, 501,    "",    "Cannot assign to variable", kjstReferenceError, 0)
+
+//RT_ERROR_MSG(JSERR_NotSafeForScripting, 502,    "",    "Object not safe for scripting", kjstError, 0)
+//RT_ERROR_MSG(JSERR_NotSafeForInitializing, 503,    "",    "Object not safe for initializing", kjstError, 0)
+//RT_ERROR_MSG(JSERR_NotSafeForCreating, 504,    "",    "Object not safe for creating", kjstError, 0)
+//RT_ERROR_MSG(JSERR_InvalidReference, 505, "", "Invalid or unqualified reference")
+//RT_ERROR_MSG(JSERR_ClassNotDefined, 506, "", "Class not defined")
+//RT_ERROR_MSG(JSERR_ComponentException, 507, "", "An exception occurred", kjstError, 0)
```

Comment:
`JSERR_None` and `JSERR_ComponentException` were apparently not used in `lib/Parser/screrror.cpp`. (All the others were.) Is it safe to comment them out or should I leave them anyway?